media/doc/Romanian translation notes.txt

```diff
@@ -1,53 +1,19 @@
 The following material is addressed (in Romanian) to Romanian translators.
 
 Copyright 2014 Ștefan Fulea
```

Comment:
```suggestion
```
I think we can strip him, since there is none of the initial documents content left over.